functests/execsboms.py

```diff
@@ -25,6 +26,8 @@ class TestSBOM(TestCase):
     Test Archivist SBOM upload/download
     """
 
+    maxDiff = None
```

Comment:
this seems to be unused?

Reply:
no - this tells the unittest framework to always output a nice diff on failure...

https://docs.python.org/3/library/unittest.html#unittest.TestCase.maxDiff